sandbox/src/mocks/bookOrDeferAppointment/requests/DeferralBookingAttemptProblem.json

```diff
@@ -0,0 +1,49 @@
+{
+  "id": "appointment",
+  "meta": {
+    "profile": [
+      "https://fhir.nhs.uk/STU3/StructureDefinition/eRS-Appointment-1",
+      "https://fhir.hl7.org.uk/STU3/StructureDefinition/CareConnect-Appointment-1"
+    ]
+  },
+  "language": "en",
+  "resourceType": "Appointment",
+  "status": "proposed",
+  "reason": [
+    {
+      "coding": [
+        {
+          "system": "https://fhir.nhs.uk/STU3/CodeSystem/eRS-ReviewReason-1",
+          "code": "BOOKING_ATTEMPT_PROBLEM"
```

Comment:
I think when supplying the "BOOKING_ATTEMPT_PROBLEM" reason you also need to provide the reference to the slot you were attempting to book to.

Reply:
Ah nope sorry I'm thinking of the SLOT_NOT_AVAILABLE case.